packages/playground/src/bots/BlockGenerateBot.ts

```diff
@@ -0,0 +1,20 @@
+import { AbstractBot } from '../AbstractBot'
+import { FoundationBot } from './FoundationBot'
+
+/**
+ * Generate a block every cycle
+ */
+export class BlockGenerateBot extends AbstractBot {
+  private static randomNodeAddress (): string {
```

Comment:
Initially thinking if move this static method to FoundationBot.

Changed my mind, I think the purpose here is to allow the random address pool to include extra injected non foundation masternodes?

Reply:
The main purpose is to allow a pool of addresses so we can split up the UTXO. Ideally when we have a full node jellyfish wallet, I would like to move the responsibility over to that instead of defid.